src/Components/Animation.js

```diff
@@ -332,7 +332,7 @@ class Animation extends React.Component {
 
     return (
       <View style={[styles.mainContainer, { height: this.animationHeight }, { width: this.animationWidth }]}>
-        {this._display(this.cuts)}
+        {!this.props.readonly && this._display(this.cuts)}
```

Comment:
Not sure we should hide the cuts in the app. I think it helps understand the movements of the players

Reply:
As on ultiplays I am on the deleting the cut

Reply:
Ok, not so important to me, let's go this way

Reply:
(Just for the record and later discution, the cuts are displayed on ultiplay at the end)

<img width="402" alt="Screenshot 2020-05-08 at 22 19 43" src="https://user-images.githubusercontent.com/3810795/81445898-13a77280-917a-11ea-91ee-b32ba7788cd6.png">